### PR TITLE
Tests and cleanups

### PR DESCRIPTION
**Pull request by ssnio**

### CODE REVIEW

File: docs/literate/examples.jl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# # Phase Slope Index method
+# # Phase Slope Index method: Notebook
```
Comment (adrhill) [inline docs/literate/examples.jl]: This will show up in the docs too.

File: src/PhaseSlopeIndex.jl [hunk 2]
```diff
@@ -102,18 +103,27 @@ function data2para(
...
     method::String,
     subave::Bool,
+    verbose::Bool
 )
-
-    # We would like to avoid transpose and copying the data!
+    # data dimension
+    if ndims(data) != 2
+        data = squeeze(data)
+        ndims(data) != 2 && throw(DimensionMismatch("data must be a 2D-array!"))
```
Comment (adrhill) [inline src/PhaseSlopeIndex.jl:111]: Won't this error always be thrown?

File: docs/literate/examples.jl (hunk 1 above)
Comment (ssnio) [reply docs/literate/examples.jl]: Thank you. You are right.

Comment (ssnio) [reply docs/literate/examples.jl]: Thank you. You are right.

File: src/PhaseSlopeIndex.jl (hunk 2 above)
Comment (ssnio) [reply src/PhaseSlopeIndex.jl:111]: Not if the squeeze has reduced it to 2D.

**Pull request merged**